recipe/meta.yaml

```
@@ -70,7 +70,8 @@ outputs:
     build:
       track_features:
         - root         # [variant and variant.startswith("root_")]
-      string: {{ variant }}_h{{ PKG_HASH }}_{{ build_number }}
+      string: {{ variant }}_flags_h{{ PKG_HASH }}_{{ build_number }}    # [with_flags]
+      string: {{ variant }}_noflags_h{{ PKG_HASH }}_{{ build_number }}  # [not with_flags]
```

I'd prefer calling this
```suggestion
      string: {{ variant }}_withcfg_h{{ PKG_HASH }}_{{ build_number }}  # [with_cfg]
      string: {{ variant }}_nocfg_h{{ PKG_HASH }}_{{ build_number }}    # [not with_cfg]
```